src/groundStation/groundStation.py

```diff
@@ -320,6 +320,60 @@ def getOptions(self):
         return self.parser.parse_args(sys.argv[1:])
 
 
+class embedCSP:
+    def __init__(self, data):
+        self.data = data
+        self.opts = options()
+        self.csp = groundStation(self.opts.getOptions())
+        # open the scheduler text file as an array of strings
+        with open('schedule.txt') as f:
+            self.cmdList = f.readlines()
+        #create an empty list, and create another list of csp objects
+        self.schedule = list()
+    
+    def embedCSP(self):
```

Comment:
Can we change the name of this method (maybe something like `getEmbededCSPData`). It's a bit confusing having it be the same name as the class

Reply:
done